docs/runtimes/application.ipynb

```diff
@@ -0,0 +1,103 @@
+{
```

Comment:
<ol><li>I would remove the <code>(a web application)</code> part from the top or change it to <code>(e.g. a web application)</code></li><li> pull at run-time is not supported yet.</li><li>API gateway can be disabled to allow free access without authentication</li></ol><p>		&gt;The deployment ensures an API Gateway</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/6036/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>